core/parachain/validator/impl/parachain_processor.cpp

```diff
@@ -1346,19 +1357,53 @@ namespace kagome::parachain {
     auto se = pm_->getStreamEngine();
     BOOST_ASSERT(se);
 
+    // TODO(turuslan): #1757, LargeStatement
+    auto message = std::make_shared<
+        network::WireMessage<network::ValidatorProtocolMessage>>(
+        network::ValidatorProtocolMessage{
+            network::StatementDistributionMessage{network::Seconded{
+                .relay_parent = relay_parent, .statement = statement}}});
+
+    std::unordered_set<network::PeerId> group_set;
```

Comment:
Please add brief comment describing that we are preparing set of peers to whom we will distribute messages